src/java.base/share/classes/sun/security/util/KnownOIDs.java

```diff
@@ -352,6 +352,9 @@
 
     // OIW secsig 1.3.14.3.*
     OIW_DES_CBC("1.3.14.3.2.7", "DES/CBC"),
+    OIW_DES_ECB("1.3.14.3.2.6", "DES/ECB", "DES"),
```

Comment:
What OID are other vendors using?

Reply:
At least BouncyCastle uses the same OIDs: http://javadox.com/org.bouncycastle/bcprov-jdk15on/1.51/org/bouncycastle/asn1/oiw/OIWObjectIdentifiers.html

Reply:
Yes I can see the OIDs. I was just wondering if other vendors are also using the OID for DES/ECB when storing a DES key. After all, this is only a key and it can can be used with all Cipher modes.

Reply:
Verified with BC: BouncyCastle uses DES/CBC OID for DES secret key in PKCS#12.

Reply:
Good. I also just noticed that the BC provider uses 1.3.14.3.2.7 as alias for KeyGenerator.DES.